bedrock/newsletter/templates/newsletter/developer.html

```diff
@@ -7,7 +7,7 @@
 {% set_lang_files "mozorg/newsletters" %}
 
 {% block page_title %}{{ _('Love the web? So do we!') }}{% endblock page_title %}
-{% block page_desc %}{{ _('Join thousands of developers like you who are learning the best of web development.') }}{% endblock %}
+{% block page_desc %}{{ _('Unlock the world of web development with our weekly Mozilla Developer Newsletter. Each edition brings you coding techniques and best practices, MDN updates, info about emerging technologies, developer tools tips, and more.') }}{% endblock %}
```

Comment:
It looks like this page is localized, so changing the string here will break translations (resulting in an English string being visible on non-English pages). This is where you'll need to use an l10n tag to stage new copy changes: https://bedrock.readthedocs.io/en/latest/l10n.html#staging-copy-changes